common/plugins/add_endpoint.go

```diff
@@ -69,6 +70,16 @@ func SetupVeth(id string, mtu int, ep *endpoint.Endpoint) (*netlink.Veth, *netli
 
 	log.Debugf("Created veth pair %s <-> %s", lxcIfName, veth.PeerName)
 
+	// Disable reverse path filter on the host side veth peer to allow
+	// container addresses to be used as source address when the linux
+	// stack performs routing.
+	args := []string{"-w", "net.ipv4.conf." + lxcIfName + ".rp_filter=0"}
+	_, err = exec.Command("sysctl", args...).CombinedOutput()
+	if err != nil {
+		return nil, nil, "", fmt.Errorf("Unable to disable rp_filter on %s: %s",
```

Comment:
s/Unable/unable.

When returning errors the error should start with small case so that when logging we could use it on a sentence. "Something bad happen: %s"